src/gesture-recognizers.js

```diff
@@ -691,42 +693,44 @@ export class PinchRecognizer extends GestureRecognizer {
   /** @override */
   onTouchStart(e) {
     const touches = e.touches;
```

Comment:
Should only stop tracking this series if touches are more than 2. If there has been one touch, `PinchRecognizer` should wait for a second touch, since practically speaking, when you pinch a phone, the touches very rarely happen simultaneously.